web_src/js/features/serviceworker.js

```diff
@@ -1,16 +1,18 @@
 const {UseServiceWorker, AppSubUrl, AppVer} = window.config;
-const cacheName = 'static-cache-v2';
+const cachePrefix = 'static-cache-v'; // actual version is set in the service worker script
 
 async function unregister() {
-  for (const registration of await navigator.serviceWorker.getRegistrations()) {
-    const serviceWorker = registration.active;
-    if (!serviceWorker) continue;
-    registration.unregister();
-  }
+  const registrations = await navigator.serviceWorker.getRegistrations();
+  await Promise.all(registrations.map((registration) => {
+    return registration.active && registration.unregister();
```

Comment:
Whatever is returned here will be awaited by `Promise.all`. In case the registration is not active, it will return `undefined` so nothing happens in that case. One could also do it in two steps with `array#filter` and `array#map` but I like this shorter version.